packages/live-share-media/src/VolumeLimiter.ts

```diff
@@ -3,59 +3,60 @@
  * Licensed under the Microsoft Live Share SDK License.
  */
 
- import { TimeInterval } from '@microsoft/live-share';
- import { IMediaPlayer } from './IMediaPlayer';
+import { TimeInterval } from '@microsoft/live-share';
+import { IMediaPlayer } from './IMediaPlayer';
 
 export enum LevelType { fixed, percentage }
-
 export class VolumeLimiter {
```

Comment:
I think we need a comment for this like other public exported classes for dev docs

Reply:
Something like "Smooth reduction of volume while the `IMediaPlayer` while ramping is active down to a set minimum and increases the volume to a set maximum when ramping ends."